playground/components.go

```diff
@@ -225,6 +234,43 @@ func (w *WebsocketProxy) Name() string {
 	return "websocket-proxy"
 }
 
+type ChainMonitor struct {
+	L1RPC            string
+	L2BlockTime      string
+	L2BuilderAddress string
+	L2RPC            string
+	ServerListenAddress string
```

Comment:
Why do you need the `ServerListenAddress` field? It seems that it complicates the component and it is not used.